client/my-sites/checkout/checkout/index.jsx

```diff
@@ -686,7 +685,6 @@ export class Checkout extends React.Component {
 			cards,
 			productsList,
 			setHeaderText,
-			showGuaranteeSeal,
 			userCountryCode,
 			isJetpackNotAtomic,
```

Comment:
We don't need `isJetpackNotAtomic ` prop too(added in #35404)

Reply:
Removed in https://github.com/Automattic/wp-calypso/pull/35728/commits/4d27869d6849e927e495261f64f6d0667da649b8